<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageBoxAsk</class>
 <widget class="QWidget" name="MessageBoxAsk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
        QWidget{
        background-color:none;
        color:none
        }
      </string>
  </property>
  <widget class="QWidget" name="MessageBox" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>448</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
          QWidget{
          background-color: #FFFFFF;
          border-radius: 8px;
          }
        </string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>448</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
            QWidget{
            background-color:#EFB92B;
            border-radius: 8px;
            }
          </string>
    </property>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>8</y>
       <width>448</width>
       <height>8</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
              QWidget{
              background-color:white;
              }
            </string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>448</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/Public/TWAIN/Resources/QMessageBox_Icon.png</normaloff>../../../Users/Public/TWAIN/Resources/QMessageBox_Icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="YesButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>231</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
            QPushButton{
            color:#0050F2;
            border-top:1px solid #DDE3ED;
            border-radius:0;
            }
          </string>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>441</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>32</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="messageBoxContent">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color:#828D9E;
                  }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>32</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="NoButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>224</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
            QPushButton{
            color:#353D45;
            border-top:1px solid #DDE3ED;
border-right:1px solid #DDE3ED;
            border-radius:0;
            }
          </string>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>YesButton</zorder>
   <zorder>NoButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
